python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -185,6 +190,12 @@ keys:
               description: |-
                 BGP AS <1-4294967295> or AS number in asdot notation "<1-65535>.<0-65535>".
                 For asdot notation in YAML inputs, the value must be put in quotes, to prevent it from being interpreted as a float number.
+      neighbor_default:
+        type: dict
+        keys:
+          send_community:
+            type: str
+            description: "`all` or a combination of `standard`, `extended` and `large`."
```

Comment:
We wanted to use valid_values here right?

Reply:
Added!